azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -267,6 +267,66 @@ func resourceStorageAccount() *schema.Resource {
 								},
 							},
 						},
+
+						"change_feed": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"retention_in_days": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntBetween(1, 146000),
```

Comment:
> 146000/365 = 400

This is 400 years worth of changes.. is there a realistic maximum value from the service team here?

Reply:
Yes. It's 400 years.

Reply:
> Minimum value is 1 day and maximum value is 146000 days (400 years).